doc/nrf/releases/release-notes-latest.rst

```diff
@@ -24,7 +24,29 @@ The Zephyr fork in |NCS| contains all commits from the upstream Zephyr repositor
 
 The following list summarizes the most important changes inherited from upstream Zephyr:
 
-*
+* LwM2M:
+
+  - Fixed a bug, where FOTA socket was not closed after the download (PULL mode).
+  - Allow to specify how early before the timeout shall Register Update be sent.
+  - Added ObjLnk resource type support.
+
+* MQTT:
+
+  - `utf8` pointer in `mqtt_utf8` struct is now const.
+  - The default `clean_session` value is now configurable with Kconfig.
+
+* OpenThread:
+  - Updated OpenThread revision up to upstream commit e653478c503d5b13207b01938fa1fa494a8b87d3.
+  - Implemented missing `enable` API function for OpenThread interface.
+  - Cleaned up OpenThread Kconfig file - OpenThread dependencies are now enabled automatically.
+  - Allow the applicaiton to register OpenThread state change callback.
+  - Reimplemented logger glue layer, for better performance.
+  - OpenThread thread priority class is now configurable.
+  - Several new Kconfig options added, customizing OT stack.
```

Comment:
Can we list the Kconfig options that were added, or are there too many?

Reply:
There were quite a few of them, IMO not worth listing them all.